qemu/tests/rng_hotplug.py

```diff
@@ -99,7 +105,14 @@ def stop_rngd(vm):
         for device in device_ids:
             unplug_rng(vm, device)
 
-    for i in xrange(repeat_times):
+    if sub_type_after_unplug:
+        error_context.context("Running sub test %s after unplug" % sub_type_after_unplug, logging.info)
+        utils_test.run_virt_sub_test(test, params, env, sub_type_after_unplug)
+
+    if sub_type_after_unplug == "shutdown":
+        return
+
+    for i in xrange(repeat_times-1):
```

Comment:
Could use `range(repeat_times)` instead ?

Reply:
No.
If use range(),also need to use range(repeate_time-1)